src/ua.c

```diff
@@ -1982,6 +2021,59 @@ struct sipevent_sock *uag_sipevent_sock(void)
 }
 
 
+static bool uri_match_transport(const struct uri *accu, enum sip_transp tp)
+{
+	struct pl pl;
+	int err;
+
+	err = msg_param_decode(&accu->params, "transport", &pl);
+	if (err)
+		return true;
+
+	return !pl_strcasecmp(&pl, sip_transp_name(tp));
+}
+
+
+static bool uri_match_af(const struct uri *accu, const struct uri *peeru)
+{
+#ifdef HAVE_INET6
```

Comment:
I am wondering if we should get rid of the `ifdef HAVE_INET6` compile time check?

Reply:
Hm, we had to disable HAVE_INET6 for one project where the customer has HW + OS linux with disabled IPv6 in kernel. With HAVE_INET6 the network initialization failed. Anyway, it should be possible to fix this in baresip.